### DISCUSSION TITLE
Meaning of stress component 0-8

### DISCUSSION THREAD

**Discussion by qiuqiubing** [Anura3D/Anura3D_OpenSource#85] [Q&A] [answered]
Dear all,
I would like to ask you two questions:
1. What's the meaning of stress component 0-8 in the results? I know that in a two-dimensional problem, 0 is horizontal stress and 4 is vertical stress. What are the other values? 
2. What is the expression for the magnitude of stress?
Thank you!

**Comment by ayerrocolom** [comment, marked as answer]
1. 0-8 components are the nine components of a 3x3 matrix (e.g., stress tensor). Component 0 is the one located in positioin (1,1); component 1 is (1,2) component 3 is (1,3); component 4 is (2,1);... and so on. Note that this notation in not decided by Anura3D but by the open-source ParaView. 
2. Magnitude is the modulus considering all components. Again, this is defined by ParaView.
